alpha/apps/kaltura/lib/kObjectCreatedHandler.php

```diff
@@ -28,46 +28,61 @@ public function shouldConsumeCreatedEvent(BaseObject $object)
 	 */
 	public function objectCreated(BaseObject $object)
 	{
-		if($object instanceof KuserKgroup)
-		{
-			$kgroup = kuserPeer::retrieveByPK($object->getKgroupId());
-			$numberOfUsersPerGroup = $kgroup->getMembersCount();
-			$kgroup->setMembersCount($numberOfUsersPerGroup+1);
-			$kgroup->save();
-			return true;
-		}
+        if ($object instanceof KuserKgroup)
+        {
+            $this->handleKuserKgroupCreated($object);
```

Comment:
to keep full backward compatibility, should return when handling specific object type